data/json/items/generic/currency.json

```diff
@@ -46,6 +46,21 @@
     "symbol": "$",
     "material": [ "copper" ]
   },
+  {
+    "type": "GENERIC",
+    "category": "currency",
+    "id": "coin_dollar_raiders",
+    "name": { "str": "Raider's coin" },
+    "description": "A one-dollar coin, featuring a portrait of an unknown, rough-looking individual.  Two rifles form a cross pattern behind them, while a mushroom cloud completes a halo in the background.  Beneath the portrait is stamped \"When in doubt, knock 'em out\".  You could flip it.",
```

Comment:
While they would probably have a currency, considering themselves legitimate government, this is a little post-apoc bandit tropey.